src/pages/api/hubspot/contacts/index.ts

```diff
@@ -11,7 +12,19 @@ export const contactResponseSchema = z.object({
 
 export function createHandler(hubspotClient: HubspotClient) {
   return async function handler(req: NextApiRequest, res: NextApiResponse) {
-    const email = req.body.email;
+    const auth = getAuth(req);
```

Comment:
I learnt today that `getCurrentUser()` can't be used outside of App router. Instead you have to get the user's session using `getAuth()` and then fetch their user from the Clerk API. This is apparently what `getCurrentUser()` does under the hook which is a bit sucky. If this becomes a performance pain we could add the data we use frequently as session claims which will make them available immediately from the session cookie. In this case adding the primary email address might make sense if we need it a lot.